hypothesis-python/src/hypothesis/internal/conjecture/engine.py

```diff
@@ -98,6 +99,9 @@ def __init__(self, test_function, settings=None, random=None, database_key=None)
         self.target_selector = TargetSelector(self.random)
 
         self.interesting_examples = {}
+        # We use call_count because there may be few possible valid_examples.
+        self.first_bug_calls = 0
```

Comment:
nit: I find these names slightly confusing. Not sure what to suggest. Maybe `{first/last}_bug_found_at` or something like that?

Also maybe set them to `None` initially? Having them start as a valid but wrong integer seems weird.